src/tasks/task1.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (arrayClothes, param, value) => {
+  const result = arrayClothes.filter(clothes => {
+    if (clothes[param] === value) return clothes;
+  });
```

Comment:
This is much better!
The only thing that needs to be improved is the return statement. 
Please read how to use `callback` function with `filter()` and update this code accordingly.